frame/assistants/feemarket-s2s-transition/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "feemarket-s2s-transition"
```

Comment:
We need to copy feemarket-s2s to the new crate, because the version v0.12.1 of Darwinia/Crab and Pangolin/Pangoro have differents feemarket dependency.
Darwinia/Crab use 
```
darwinia-fee-market    = { git = "https://github.com/darwinia-network/darwinia-common.git", branch = "darwinia-v0.12.1" }
```
Pangolin/Pangoro use

```
darwinia-fee-market    = { git = "https://github.com/darwinia-network/darwinia-common.git", tag = "darwinia-v2.8.7" }
```

The next version will be to delete this crate. because will use the same dependency